task-3/task-3.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+func main() {
+	var n int
+	fmt.Println("Введите n")
+	fmt.Scan(&n)
+	a := make([]int, n)
+	temp := 0
+	for i := 0; i < n; i++ {
+		a[i] = rand.Intn(10)
+	}
+	for i := 0; i < n; i++ {
+		fmt.Print(a[i])
+	}
+	println(" ")
```

Comment:
?)
если нужно перенести строку, то можно и так
```suggestion
	println()
```